### PR TITLE
CSS Changes

### PR DESCRIPTION
**Pull request by solomonjoseph**
1. Redesigned login page to create more modern and elegant feel
2. Fixed alignments for sign up and hkn elections pages

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,14 +4,36 @@ verify_ssl = true
 name = 'pypi'
 
 [packages]
-django = "==2.2.7"
```
Comment (alexander-zw) [inline Pipfile]: Why did you change the pipfile?

File: hknweb/templates/registration/login.html [hunk 2]
```diff
@@ -1,34 +1,148 @@
 {% extends "base.html" %}
 {% load static %}
+{% block header %}
+<style>
```
Comment (alexander-zw) [inline hknweb/templates/registration/login.html]: I think it would be better to have these in a style.css file

File: Pipfile (hunk 1 above)
Comment (solomonjoseph) [reply Pipfile]: I didn't mean to change the pipfile

File: hknweb/templates/registration/login.html (hunk 2 above)
Comment (solomonjoseph) [reply hknweb/templates/registration/login.html]: Resolved

File: Pipfile (hunk 1 above)
Comment (GeoHutch) [reply Pipfile]: https://www.atlassian.com/git/tutorials/rewriting-history

Here's some info on rebasing bad changes that made it into a PR.

**Comment by ochan1** [comment]
Old PR, and fork was deleted
Decision to Close the PR
